configs/cgroup.go

```diff
@@ -87,6 +87,9 @@ type Cgroup struct {
 	// Whether to disable OOM Killer
 	OomKillDisable bool `json:"oom_kill_disable"`
 
+	// Tuning swappiness behaviour per cgroup
+	MemSwappiness int64 `json:"swappiness"`
```

Comment:
Can we call it `MemorySwappiness` for consistency? and move it to another `Memory..` fields?
Also as I see we don't use short names in json, so should be `memory_swappiness`